include/geode/inspector/topology/brep_topology.h

```diff
@@ -53,55 +53,68 @@ namespace geode
 
         bool brep_meshed_components_are_linked_to_a_unique_vertex() const;
 
-        index_t nb_corners_not_linked_to_a_unique_vertex() const;
+        index_t nb_corners_not_linked_to_a_unique_vertex(
+            bool verbose = false ) const;
```

Comment:
Like I said, I really do not like default parameters... I also think adding a verbose parameter everywhere is too much...

How about having the verbose parameter in the class constructor only? You could always overload the constructor if you like and default the verbose value as member initialization.